client/src/app/components/tool-container/tool-header/tool-header.component.scss

```diff
@@ -0,0 +1,26 @@
+@import "../../../../assets/sass/variables";
+
+header {
+    #revisionSelect {
+        color: #000000;
+        margin-right: 40px;
+        width: 100px;
+    }
+
+    button {
+        width: 8%;
+    }
+
+    #guiJsonButtons {
```

Comment:
Also, `header` is a standard DOM element for which we shouldn't assume a particular styling.